library/src/main/res/layout/voucher_checkout_view.xml

```diff
@@ -25,6 +29,6 @@
             android:layout_height="wrap_content"
             android:layout_marginTop="10dp"
             android:enabled="false"
-            android:textColor="@android:color/white" />
+            style="@style/VoucherValidateButton" />
     </LinearLayout>
 </merge>
```

Comment:
Can we change the name of the file to start from 'view_' so we will have view_voucher_checkout